engine/server/server.h

```diff
@@ -445,6 +445,10 @@ extern	convar_t		*sv_hostmap;
 extern	convar_t		*sv_validate_changelevel;
 extern	convar_t		*public_server;
 
+// 0.19.2 stuff
```

Comment:
You don't need to comment "it's from 0.19.2".

Just modernize code a bit and it's ok for new engine.